### PR TITLE
MDP-199: Modal update

### PR DESCRIPTION
**Pull request by OlehVynnyk**

### CODE REVIEW

**Review by slavomircesla — CHANGES_REQUESTED** [review]
components/Popup/Button.tsx
Change /rounded-8/ to /rounded-12/

File: src/components/Popup/InfoMessage.tsx [hunk 1]
```diff
@@ -5,5 +5,10 @@ interface IProps {
 }
 
 export const InfoMessage: FC<IProps> = ({ text }) => (
-  <div className="mt-20 mr-48 ml-24 font-lato text-16">{text}</div>
+  <div
+    style={{ lineHeight: "1.38" }}
+    className="mt-16 mr-48 ml-24 font-lato text-16"
```
Comment (slavomircesla) [inline src/components/Popup/InfoMessage.tsx]: only change to mt-20

**Review by slavomircesla — DISMISSED** [review]

File: src/components/Popup/CloseButton.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import React from "react";
+import { Icon } from "../Icon";
+import iconTimes from "../../static/svg/icon-times.svg";
```
Comment (garySiladi) [inline src/components/Popup/CloseButton.tsx]: this should be named closeIcon

File: src/components/Popup/Storybook/RegistrationPopup.tsx [hunk 3]
```diff
@@ -5,16 +5,18 @@ import { InfoMessage } from "../InfoMessage";
...
 import { Popup } from "../Popup";
+import { PaymentInfo } from "../PaymentInfo";
 
 export const RegistrationPopup = () => (
   <Popup>
     <div>
       <CloseButton />
       <Message text="Záväzná registrácia" />
       <InfoMessage text="BABY CHAT - 15:30, 28.január 2019" />
+      <PaymentInfo text="Cena arteterapie - 10 € " textColor="black" />
```
Comment (garySiladi) [inline src/components/Popup/Storybook/RegistrationPopup.tsx]: remove empty space in the end of the text string

File: src/components/Schedule/ProgressBar.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+interface IProps {
+  actual: number;
+  max: number;
+}
+
+export const ProgressBar: FC<IProps> = ({ actual, max }) => {
+  if (actual === 0) {
```
Comment (garySiladi) [inline src/components/Schedule/ProgressBar.tsx]: don't we want to render a progress bar, if the reservations are empty?

File: src/components/Schedule/ProgressBar.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    return null;
+  }
+  return (
+    <div
+      style={{ width: (actual / max) * 100 + "%" }}
+      className="border-b border-2 border-black absolute"
```
Comment (garySiladi) [inline src/components/Schedule/ProgressBar.tsx]: style this by giving background color and don't use absolute

File: src/components/Schedule/ProgressBar.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  actual: number;
+  max: number;
+}
+
+export const ProgressBar: FC<IProps> = ({ actual, max }) => {
+  if (actual === 0) {
+    return null;
+  }
+  return (
+    <div
```
Comment (garySiladi) [inline src/components/Schedule/ProgressBar.tsx]: don't we need the grey line behind the black one as well?

**Review by garySiladi — CHANGES_REQUESTED** [review]

**Comment by garySiladi** [comment]
change the branching for this, it contains unrelated files

**Pull request closed without merging**